chatlogger.lua

```diff
@@ -12,33 +12,39 @@ function OnStartup()
 		sBotName = SetMan.GetString( 21 ) or "PtokaX",
 		sProfiles = "012",		-- No history for commands from users with profiles
 		sLogsPath = "/www/ChatLogs/",
+		sTimeFormat = "[%I:%M:%S %p] ",
```

Comment:
No! `OnStartup` is executed just once, while we want to compute `os.date` every time before writing to the file.